qa/workunits/ceph-helpers.sh

```diff
@@ -608,7 +625,7 @@ function activate_osd() {
         --mark-init=none \
         $osd_data || return 1
 
-    [ "$id" = "$(cat $osd_data/whoami)" ] || return 1
+    [ "$id" -eq "$(cat $osd_data/whoami)" ] || return 1
```

Comment:
@dachary 
I'll remove it, 
Although it is nor really a cosmetic fix. `=` is a string compair, and I've seen tools output with an extra ` ` so that `3 ` != 3.  `-eq` does not suffer from that. Especially is there are `""`'s around.